datalad/interface/run.py

```diff
@@ -458,12 +458,22 @@ def normalize_command(command):
     """Convert `command` to the string representation.
     """
     if isinstance(command, list):
+        command = list(map(assure_unicode, command))
```

Comment:
all PY2 kids would appreciate `from six.moves import map` to be like their big PY3 brother

Reply:
sorry, I'm bad about only thinking of py2 once I start to see failures.  Will adjust.